component/component/vshn_postgres.jsonnet

```diff
@@ -23,6 +23,7 @@ local certificateSecretName = 'tls-certificate';
 local serviceNameLabelKey = 'appcat.vshn.io/servicename';
 local serviceNamespaceLabelKey = 'appcat.vshn.io/claim-namespace';
 local slaLabelKey = 'appcat.vshn.io/sla';
+local serviceCLaimNameLabelKey = 'appcat.vshn.io/claim-name';
```

Comment:
This variable is not used anywhere and also contains a typo (serviceC**L**aimNameLabelKey)

Reply:
Hmmm... I think i missed those when rebasing to master.